### ISSUE TITLE
Participate.js should read the config files for master URI

### ISSUE THREAD

**Issue by Bill-Keuntje** [rchain-community/rgov#187]
> Yes, a configuration file would be appropriate.

Note that a configuration file doesn't get `import`ed. `import` is for source code; i.e. design-time materials.

A configuration file would get read in at runtime.

(Another alternative would be an environment variable, as advocated by https://12factor.net/config )

_Originally posted by @dckc in https://github.com/rchain-community/rgov/issues/185#issuecomment-853575240_

**Comment by Bill-Kunj** [comment]
> > @Bill-Kunj does hardcoding the values work?
> 
> @David405 I don't think so.
> @jimscarver What do we need for rv2021?

@dckc Please confirm that I am correct that hard-coding is not the solution here?

**Comment by dckc** [comment]
I agree: the master URI is not a design-time constant -- it changes at runtime depending on the chain -- so it doesn't belong in the source code.

**Comment by jimscarver** [comment]
the legacy code hard codes the uri's but that not how we want to do it. For 2021 if it works this week with the uris hard coded that is good enough. Taking the uris out of the code is the priority once it works at all.
